### PR TITLE
LPS-159508 Set the exported file name proper

### PR DESCRIPTION
**Pull request by sontruongces**
Hi team,

This is the fix for the [LPS-159508](https://issues.liferay.com/browse/LPS-159508). The original fix has passed my review at sontruongcces#26. Please help me to review and give feedback.
cc:@quanhuynhces

Thanks!

### CODE REVIEW

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java [hunk 1]
```diff
@@ -180,6 +180,8 @@ private void _exportAll(
 
 		String languageId = themeDisplay.getLanguageId();
 
+		String portletName = themeDisplay.getPpid();
```
Comment (drewbrokke) [inline modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java:183]: This variable should be defined right before it is used. In fact, no variable is needed, as the `if` statement can just be condensed to this: 
```java
Objects.equals(ConfigurationAdminPortletKeys.INSTANCE_SETTINGS, themeDisplay.getPpid())
```

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java (hunk 1 above)
Comment (quanhuynhces) [reply modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java]: @drewbrokke I updated the code. Thank you.

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java [hunk 2]
```diff
@@ -233,7 +234,17 @@ else if (configurationModel.hasConfiguration()) {
...
+
+		if (Objects.equals(
+				ConfigurationAdminPortletKeys.INSTANCE_SETTINGS,
+				themeDisplay.getPpid())) {
+
+			fileName = "liferay-instance-settings.zip";
+		}
+		else {
+			fileName = "liferay-system-settings.zip";
+		}
```
Comment (drewbrokke) [inline modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java:247]: I think we need to also cover the case for exporting from site settings as well.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java (hunk 2 above)
Comment (quanhuynhces) [reply modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java:250]: Thank you @drewbrokke. I updated the code

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java [hunk 3]
```diff
@@ -233,7 +234,23 @@ else if (configurationModel.hasConfiguration()) {
...
+		else if (Objects.equals(
+					ConfigurationAdminPortletKeys.SYSTEM_SETTINGS,
+					themeDisplay.getPpid())) {
+
+			fileName = "liferay-system-settings.zip";
+		}
+		else {
+			fileName = "liferay-site-settings.zip";
+		}
```
Comment (drewbrokke) [inline modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java:253]: This works, but it should follow a logical order. Let's start with `system` scope, then check `instance` scope, then check `site` scope.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java (hunk 3 above)
Comment (quanhuynhces) [reply modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java:253]: @drewbrokke Thank you. I updated the code

Comment (drewbrokke) [reply modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/portlet/action/ExportConfigurationMVCResourceCommand.java:253]: Thanks @quanhuynhces!

**Comment by drewbrokke** [comment]
ci:forward

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**